web/channels/retro_channel.ex

```diff
@@ -79,16 +79,28 @@ defmodule RemoteRetro.RetroChannel do
 
   def handle_in("submit_vote", %{"ideaId" => idea_id, "userId" => user_id}, socket) do
     retro_id = socket.assigns.retro_id
-    idea_query = from i in Idea, where: i.id == ^idea_id
-    participation_query = from p in Participation, where: p.user_id == ^user_id and p.retro_id == ^retro_id
+    idea_vote_count = Repo.get(Idea, idea_id).vote_count
+    particip_vote_count = Repo.get_by(Participation, user_id: user_id, retro_id: retro_id).vote_count
 
-    {:ok, %{idea: {1, [updated_idea]}, participation: {1, [_updated_participation]}}} =
-      Multi.new
-      |> Multi.update_all(:idea, idea_query, [inc: [vote_count: 1]], returning: true)
-      |> Multi.update_all(:participation, participation_query, [inc: [vote_count: 1]], returning: true)
-      |> Repo.transaction
+    idea_changeset =
+      Repo.get(Idea, idea_id)
+      |> Idea.changeset(%{vote_count: idea_vote_count + 1})
+
+    participation_changeset =
+      Repo.get_by(Participation, user_id: user_id, retro_id: retro_id)
+      |> Participation.changeset(%{vote_count: particip_vote_count + 1})
+
+   %{valid?: is_participation_valid} = participation_changeset
```

Comment:
rather than destructuring here, i think (think) it would be cleaner to just reference the the `valid?` property using dot notation in the conditional below, as that's the only place the `is_participation_valid` variable is leveraged